src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/MediaLegalApplicationController.java

```diff
@@ -0,0 +1,128 @@
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.hmcts.reform.pip.account.management.authentication.roles.IsAdmin;
+import uk.gov.hmcts.reform.pip.account.management.model.MediaAndLegalApplication;
+import uk.gov.hmcts.reform.pip.account.management.model.MediaAndLegalApplicationDto;
+import uk.gov.hmcts.reform.pip.account.management.model.MediaLegalApplicationStatus;
+import uk.gov.hmcts.reform.pip.account.management.service.MediaLegalApplicationService;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.http.MediaType.MULTIPART_FORM_DATA_VALUE;
+
+@RestController
+@Api(tags = "Account Management - API for managing media & orphaned legal professional applications")
+@IsAdmin
+@RequestMapping("/application")
+public class MediaLegalApplicationController {
+
+    private final MediaLegalApplicationService mediaLegalApplicationService;
+
+    private static final String NOT_AUTHORIZED_MESSAGE = "User has not been authorized";
+
+    @Autowired
+    public MediaLegalApplicationController(MediaLegalApplicationService mediaLegalApplicationService) {
+        this.mediaLegalApplicationService = mediaLegalApplicationService;
+    }
+
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "List<{MediaAndLegalApplication}>"),
+        @ApiResponse(code = 403, message = NOT_AUTHORIZED_MESSAGE),
+    })
+    @ApiOperation("Get all applications")
+    @GetMapping(produces = APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<MediaAndLegalApplication>> getApplications() {
+        return ResponseEntity.ok(mediaLegalApplicationService.getApplications());
+    }
+
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "List<{MediaAndLegalApplication}>"),
+        @ApiResponse(code = 403, message = NOT_AUTHORIZED_MESSAGE),
+    })
+    @ApiOperation("Get all application by the status")
+    @GetMapping(value = "/status/{status}", produces = APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<MediaAndLegalApplication>> getApplicationsByStatus(
+        @PathVariable MediaLegalApplicationStatus status) {
+        return ResponseEntity.ok(mediaLegalApplicationService.getApplicationsByStatus(status));
+    }
+
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "{MediaAndLegalApplication}"),
+        @ApiResponse(code = 403, message = NOT_AUTHORIZED_MESSAGE),
+        @ApiResponse(code = 404, message = "No media application found with id: {id}"),
```

Comment:
while it looks more readable now youve provided the id, it still doesnt match your error message that you return in src/main/java/uk/gov/hmcts/reform/pip/account/management/service/MediaLegalApplicationService.java, this is where your 404 comes from

Reply:
but im fine with just leaving this for now